index.html

```diff
@@ -6,6 +6,12 @@
   <meta charset="UTF-8">
   <title>Free for developers</title>
   <link rel="stylesheet" href="//unpkg.com/docsify/themes/vue.css">
+  <link 
+      rel="stylesheet"
```

Comment:
Very nice. I wonder if the style sheet above is also on jsdeliver? Faster to use one host if possible

Reply:
updated